win32ss/gdi/eng/stretchblt.c

```diff
@@ -397,10 +507,127 @@ IntEngStretchBlt(SURFOBJ *psoDest,
     /* Sanity check */
     ASSERT(IS_VALID_ROP4(Rop4));
 
+    cxSrc = SourceRect->right - SourceRect->left;
+    cySrc = SourceRect->bottom - SourceRect->top;
+    cxDest = DestRect->right - DestRect->left;
+    cyDest = DestRect->bottom - DestRect->top;
+
+    DPRINT("SourceRect is (%d,%d)-(%d,%d) and DestRect is (%d,%d)-(%d,%d)\n",
+           DestRect->left, DestRect->top, DestRect->right, DestRect->bottom,
+           SourceRect->left, SourceRect->top, SourceRect->right, SourceRect->bottom);
+
+    DPRINT("cxDest(%d) cyDest(%d) cxSrc(%d) cySrc(%d)\n",
+           cxDest, cyDest, cxSrc, cySrc);
+
+    /* Check if source and dest size are equal in size */
+    if ((abs(cxDest) == abs(cxSrc)) && (abs(cyDest) == abs(cySrc)))
+    {
+        /* if cxSrc < 0 and cxDest <0 (both negative, we change both their signs  */
+        /* because the outcome is the same as if they were both positive and we   */
+        /* reverse both their top and bottom values                               */
```

Comment:
```suggestion
        /* if cxSrc < 0 and cxDest <0 (both negative, we change both their signs
         * because the outcome is the same as if they were both positive and we
         * reverse both their top and bottom values */
```